R/Random_SurnameName_Sample.R

```diff
@@ -0,0 +1,4 @@
+# Taking a random sample of names from census frequent names.
+random_surname_sample <- Names_2010Census[sample(nrow(Names_2010Census), 20), ]
```

Comment:
What is this script doing? How does it have access to `Names_2010Census`? I think this is leading to error in the CMD check.

One way around this is to import `wru` and use the dataset from there.